README.md

```diff
@@ -1,3 +1,5 @@
 # WhatTravel
 An Android travel app for fun
+This App is using google places api
```

Comment:
you can click the merge pull request button, and it will automatically merge the code